examples/inventory.js

```diff
@@ -132,7 +132,7 @@ function craftItem (name, amount) {
   amount = parseInt(amount, 10)
   const item = require('minecraft-data')(bot.version).findItemOrBlockByName(name)
   const craftingTable = bot.findBlock({
-    matching: 58
+    matching: require('minecraft-data')(bot.version).blocksByName.crafting_table.id
```

Comment:
Probably move this to be next to the "item" declaration variable, for readability.
```js
const mcData = require('minecraft-data')(bot.version)

const item = mcData.findItemOrBlockByName(name)
const craftingTableID = mcData.blocksByName.crafting_table.id

const craftingTable = bot.findBlock({
  matching: craftingTableID
})
```